setup.cfg

```diff
@@ -1,6 +1,88 @@
-[metadata]
-description-file = README.md
+[aliases]
+dists = clean --all sdist bdist_wheel
+
+[bdist_wheel]
+universal = 1
 
 [flake8]
 max-line-length = 100
 exclude = .git,.hg,.svn,test,setup.py,__pycache__
+
+[metadata]
+name = ansible-lint
+url = https://github.com/ansible/ansible-lint
+project_urls =
+    Bug Tracker = https://github.com/ansible/ansible-lint/issues
+    CI: Travis = https://travis-ci.org/ansible/ansible-lint
+    Code of Conduct = https://docs.ansible.com/ansible/latest/community/code_of_conduct.html
+    Documentation = https://github.com/ansible/ansible-lint/blob/master/RULE_DOCS.md
+    Mailing lists = https://docs.ansible.com/ansible/latest/community/communication.html#mailing-list-information
+    Source Code = https://github.com/ansible/ansible-lint
+description = Checks playbooks for practices and behaviour that could potentially be improved
+long_description = file: README.md  ### FIXME: README.rst or mime type
```

Comment:
@gundalow we need to decide whether we want to keep md or switch to rst. rst is native to PyPI and has reacher feature set, so it'd be better.
If not, I'll need to add a mime type option to support md.

Reply:
Personally I prefer RST. We've standardized on that in Ansible. I'll convert the `README`